beacon-chain/light-client-network.md

```diff
@@ -120,6 +125,9 @@ We define the following constants which are used in the various data type defini
 ```py
 # Maximum number of `LightClientUpdate` instances in a single request
 MAX_REQUEST_LIGHT_CLIENT_UPDATES = 2**7  # = 128
+
+# Maximum number of `HistoricalSummary` records
+HISTORICAL_ROOTS_LIMIT = 2**24  # = 16,777,216
```

Comment:
See comment below, but this doesn't need to be redefined and can be referenced from consensus spec.